inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -67,40 +67,46 @@ <h3 class="text-center header-text">
       </holding-pen-dashboard-item>
 
     </div>
-    
-    {%- if config.get('FEATURE_FLAG_ENABLE_PUBLISHER_HARVESTS_CARD', False) %}
-    <div class="col-md-3 col-sm-6">	
-      <holding-pen-dashboard-item	
-        template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"	
-        filter_string="data_type=hep&q=-metadata.acquisition_source.source:arXiv AND metadata.acquisition_source.method:hepcrawl"
-        secondary_filter="status"	
-        section_title="publisher harvests">	
-      </holding-pen-dashboard-item>	
+
+    <div class="col-md-3 col-sm-6">
+        <holding-pen-dashboard-item
+          template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
+          filter_string="source=arXiv&workflow_name=HEP&is-update=true"
+          secondary_filter="status"
+          section_title="arxiv updates">
+        </holding-pen-dashboard-item>
     </div>
-    {%- endif %}
 
     <div class="col-md-3 col-sm-6">
       <div class="overview-item">
-
         <holding-pen-dashboard-item
           template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
           filter_string="source=submitter&workflow_name=HEP"
           secondary_filter="status"
           section_title="New Literature Submissions">
         </holding-pen-dashboard-item>
-
       </div>
     </div>
-
+    
+    {%- if config.get('FEATURE_FLAG_ENABLE_PUBLISHER_HARVESTS_CARD', False) %}
```

Comment:
not sure about using harvests card feature flag for also updates card. 

@drjova are they going be shown on `prod` at the same time? or is it likely that we will want to show one but not the other?